src/apphosting/repo.ts

```diff
@@ -151,7 +151,7 @@ async function createFullyInstalledConnection(
 
   while (conn.installationState.stage !== "COMPLETE") {
     utils.logBullet("Install the Cloud Build GitHub app to enable access to GitHub repositories");
-    const targetUri = conn.installationState.actionUri.replace("install_v2", "direct_install_v2");
+    const targetUri = conn.installationState.actionUri;
```

Comment:
this is a no-op for dev connect. This was a result of some cloud build repo copy pasta.